Gemfile.lock

```diff
@@ -1,35 +1,37 @@
 PATH
   remote: .
   specs:
-    contentstack (0.6.3)
-      activesupport (~> 3.2)
+    contentstack (0.6.4)
+      activesupport (>= 3.2)
       contentstack_utils (~> 1.0)
```

Comment:
Do we need to update the Util package to 1.1.3.2 ?
https://rubygems.org/gems/contentstack_utils/versions/1.1.3.2